posix.mak

```diff
@@ -170,7 +170,7 @@ PACKAGE_std = array ascii base64 bigint bitmanip compiler complex concurrency \
   outbuffer parallelism path process random signals socket socketstream stdint \
   stdio stdiobase stream string system traits typecons typetuple uni \
   uri utf uuid variant xml zip zlib
-PACKAGE_std_experimental = typecons
+PACKAGE_std_experimental = checkedint typecons
```

Comment:
I was glad I don't need to modify other places as well :o). Anyhow I made the changes to winNN.mak as well.